src/main/java/org/broadinstitute/hellbender/tools/exome/GetBayesianHetCoverage.java

```diff
@@ -0,0 +1,300 @@
+                normalHetPulldown = normalHetPulldownCalculator.getHetPulldown(normalBamFile, hetCallingStringency);
+
+                logger.info("Writing Het pulldown from normal reads to " + normalHetOutputFile.toString());
+                normalHetPulldown.write(normalHetOutputFile);
+
+                break;
+
+            case TUMOR_ONLY:
+
+                tumorHetPulldownCalculator = new BayesianHetPulldownCalculator(REFERENCE_ARGUMENTS.getReferenceFile(),
+                        IntervalList.fromFile(snpFile), minimumMappingQuality, minimumBaseQuality, readDepthThreshold,
+                        VALIDATION_STRINGENCY, errorProbabilityAdjustmentFactor);
+                tumorHetPulldownCalculator.useHeterogeneousHetPrior(minimumAbnormalFraction, maximumAbnormalFraction, maximumCopyNumber,
+                        quadratureOrder);
+
+                logger.info("Calculating the Het pulldown from the tumor BAM file using the HETEROGENEOUS prior...");
+                tumorHetPulldown = tumorHetPulldownCalculator.getHetPulldown(tumorBamFile, hetCallingStringency);
+
+                logger.info("Writing Het pulldown from tumor reads to " + tumorHetOutputFile.toString());
+                tumorHetPulldown.write(tumorHetOutputFile);
+
+                break;
+
+            case NORMAL_TUMOR:
+
+                normalHetPulldownCalculator = new BayesianHetPulldownCalculator(REFERENCE_ARGUMENTS.getReferenceFile(),
+                        IntervalList.fromFile(snpFile), minimumMappingQuality, minimumBaseQuality, readDepthThreshold,
+                        VALIDATION_STRINGENCY, errorProbabilityAdjustmentFactor);
+
+                logger.info("Calculating the Het pulldown from the normal BAM file using the BALANCED prior...");
+                normalHetPulldown = normalHetPulldownCalculator.getHetPulldown(normalBamFile, hetCallingStringency);
+
+                logger.info("Writing Het pulldown from normal reads to " + normalHetOutputFile.toString());
+                normalHetPulldown.write(normalHetOutputFile);
+
+                tumorHetPulldownCalculator = new BayesianHetPulldownCalculator(REFERENCE_ARGUMENTS.getReferenceFile(),
+                        normalHetPulldown.getIntervals(), minimumMappingQuality, minimumBaseQuality, readDepthThreshold,
+                        VALIDATION_STRINGENCY, errorProbabilityAdjustmentFactor);
+
+                logger.info("Calculating the Het pulldown from the tumor BAM file on Hets detected in the normal BAM file...");
+                tumorHetPulldown = tumorHetPulldownCalculator.getTumorHetPulldownFromNormalPulldown(tumorBamFile,
+                        normalHetPulldown);
+
+                logger.info("Writing Het pulldown from tumor reads to " + tumorHetOutputFile.toString());
+                tumorHetPulldown.write(tumorHetOutputFile);
+
+                break;
+
+            case NONE:
+                break;
+
+            default:
+                break;
```

Comment:
Instead of breaking on the default I would throw an user-exception:

``` 
throw new UserException.BadArgumentValue(JOB_TYPE_ARG_LONG_NAME, String.format("unsupported job-type %s", jobType));
```

Please correct compilation errors.